services/docker-compose.yml

```diff
@@ -232,6 +232,7 @@ services:
       - TRACING_THRIFT_COMPACT_ENDPOINT=${TRACING_THRIFT_COMPACT_ENDPOINT}
       - DYNAMIC_SIDECAR_PROMETHEUS_SERVICE_LABELS=${DYNAMIC_SIDECAR_PROMETHEUS_SERVICE_LABELS}
       - DYNAMIC_SIDECAR_PROMETHEUS_MONITORING_NETWORKS=${DYNAMIC_SIDECAR_PROMETHEUS_MONITORING_NETWORKS}
+      - DIRECTOR_V2_DYNAMIC_SCHEDULER_IGNORE_SERVICES_SHUTDOWN_WHEN_CREDITS_LIMIT_REACHED=${DIRECTOR_V2_DYNAMIC_SCHEDULER_IGNORE_SERVICES_SHUTDOWN_WHEN_CREDITS_LIMIT_REACHED}
```

Comment:
where do you test that ENV? I did not see that one

Reply:
I've added a test for this one